ssr/render.ts

```diff
@@ -14,7 +14,15 @@ const PREFERRED_LOCALE = {
   zh: "zh-CN",
 };
 
-function htmlEscape(s) {
+// We should use the language tag (e.g. "zh-Hans") instead of the locale.
+// This is a map of locale => language tag.
+// See https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry
+const LANGUAGE_TAGS = new Map([
+  ["zh-CN", "zh-Hans"],
+  ["zh-TW", "zh-Hant"],
+]);
```

Comment:
Can we use a simple frozen object instead, and add the region?

```suggestion
const LANGUAGE_TAGS = Object.freeze({
  'zh-CN': 'zh-Hans-CN',
  'zh-TW': 'zh-Hant-TW',
});
```

Reply:
I'm not sure should we just use `zh-Hans` for Simplified Chinese (zh-CN) and `zh-Hant` for traditional Chinese (zh-TW) without the region code (`CN` and `TW`). As we are using the shortest language tags that won't be duplicated.

Reply:
You're right, using `zh-Hans` and `zh-Hant` would be consistent with using `pt` for `pt-BR`.